vizro-core/src/vizro/models/_components/ag_grid.py

```diff
@@ -111,7 +111,7 @@ def build(self):
                 # here) must have the same setting as the object that is built by the on-page-load mechanism using
                 # with the user settings and rendered finally. Otherwise the grid is not rendered correctly.
                 # Hence be careful when editing the line below.
-                html.Div(self.__call__(data_frame=pd.DataFrame()), id=self.id, className="table-container"),
+                html.Div(self.__call__(), id=self.id, className="table-container"),
```

Comment:
Should we apply the same changes in the `Table.__call__` as well?

Reply:
This is already done in a previous PR 💪